adoublef/typhoon-awe
# feat/iam module

- add type helpers
- add schema
- iam domain objects defined
- test helper for adding new user
- get profile helper happy path tested
- add test for invalid credentials
- update optionals to nullish
- helper to panic
- update deps
- add github oauth client
- update sql statement to check against login
- add modules for kv store
- update deps
- iam routes
- app for iam service
- helpers for storing profile in kv sessions
- update iam deps
- top-level middleware
- helpers for loading env vars
- intro to oauth
- iam middleware entry
- update types readme with 2nd ref
- add env type to handlers
- move index to iam
- update html entrypoint directory
- update deps
- rename main entry for application
- update github oauth client
- update task file
- update with local env
- udpate iam service
- temp rm settings page
- dummy words for docs
- apply schema to turso
